test/Streamly/Test/Prelude.hs

```diff
@@ -138,3 +226,26 @@ mixedOpsAheadly =
                     return (x11 + y11 + z11)
                 return (x1 + y1 + z1)
         return (x + y + z)
+
+main :: IO ()
+main = hspec $ H.parallel $ do
+    describe "Filtering" $ do
+        it "takeByTime" (testTakeByTime `shouldReturn` True)
+        it "dropByTime" (testDropByTime `shouldReturn` True)
+
+    describe "From Generators" $ do
+        prop "unfold" unfold
+        prop "unfold0" unfold0
+
+    it "fromCallback" $ testFromCallback `shouldReturn` (50*101)
```

Comment:
Shouldn't these go in Prelude.Serial tests instead? This module was supposed to be for things that cannot be categorized anywhere else.